awscli/clidriver.py

```diff
@@ -425,6 +425,8 @@ def __init__(self, name, parent_name, operation_caller,
         self._lineage = [self]
         self._operation_model = operation_model
         self._session = session
+        if operation_model and operation_model.deprecated:
```

Comment:
Remind me again when `operation_model` can be a False value?

Reply:
The majority of unit tests for this class pass `{}` or `None`. In practice this will never be a False value. Would you rather all the tests be updated to pass a mock?

Reply:
That would be better. I would prefer a mock that uses the `OperationModel` class as the value to `spec`.

Reply:
Sounds good, I'll update the tests.